### ISSUE TITLE
[Feature request] Support multiple wordlists

### ISSUE THREAD

**Issue by SecT0uch** [ffuf/ffuf#55]
Would it be possible to support multiple wordlists, like [wfuzz does](https://wfuzz.readthedocs.io/en/latest/user/getting.html#multiple-payloads) ?

The syntax is  `-w wordlist1.txt -w worlist2.txt` with the keywords `FUZZ`, `FUZ2Z`, `FUZnZ`..

**Comment by joohoi** [comment]
Yeah, this is on my TODO list. I'm thinking of giving the user the ability to choose the keywords at the same time. As an example:

```
ffuf -w "HOST:/path/to/host_wordlist" -w "PARAMS:/path/to/parameter_wordlist" -u https://HOSTS/path/to/content.php?param=PARAMS
```

**Comment by joohoi** [comment]
Implemented in #79

**Comment by Techbrunch** [comment]
FYI, The final implementation is not the one mentionned in the previous comment:

```
Wordlist file path and (optional) custom fuzz keyword, using colon as delimiter. Use file path '-' to read from standard input. Can be supplied multiple times. Format: '/path/to/wordlist:KEYWORD'
```

and not `KEYWORD:/path/to/wordlist`
